client/lib/domains/utils/is-recently-registered.js

```diff
@@ -1,5 +1,8 @@
 import moment from 'moment';
 
-export function isRecentlyRegistered( registrationDate ) {
-	return moment.utc( registrationDate ).isAfter( moment.utc().subtract( 30, 'minutes' ) );
+export function isRecentlyRegistered( registrationDate, minutes = 30 ) {
```

Comment:
I'm not sure if we should define a default value since we always provide `minutes` when we call that function.

Reply:
`isRecentlyRegistered` existed before and is called in two other places as well, hence the default.

Reply:
Gotcha, I didn't pay attention to that 👍.